src/util/vfs/vfs-zip.c

```diff
@@ -597,10 +597,12 @@ struct VFile* _vdzOpenFile(struct VDir* vd, const char* path, int mode) {
 	UNUSED(mode);
 	struct VDirZip* vdz = (struct VDirZip*) vd;
 
+#ifndef _MSC_VER
 	if ((mode & O_ACCMODE) != O_RDONLY) {
```

Comment:
Hmm. Maybe this is the solution? https://github.com/ArtanisCV/STLport-Visual-Studio/blob/master/src/details/fstream_win32io.cpp#L39